projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.html

```diff
@@ -17,30 +17,78 @@
+        <ng-container *ngIf="title.type === 'bf:Title'">
+          <h1>{{ title._text }}</h1>
+        </ng-container>
+      </ng-container>
+    </ng-container>
   </header>
   <section>
     <dl class="row mb-0">
+
+      <!-- ALTERNATIVE GRAPHICS -->
+      <ng-container *ngIf="record.metadata.ui_title_altgr">
+        <ng-container *ngFor="let altgr_title of record.metadata.ui_title_altgr; first as isFirst">
+          <dt class="col-sm-4 offset-sm-2 offset-md-0">
+            <span class="label-title" *ngIf="isFirst; else elseBlock" translate>Title</span>
+            <ng-template #elseBlock>&nbsp;</ng-template>
+          </dt>
+          <dd class="col-sm-6 col-md-8 mb-0">
+            {{ altgr_title | titlecase }}
+          </dd>
+        </ng-container>
+      </ng-container>
+
+      <!-- VARIANT TITLE -->
+      <ng-container *ngIf="record.metadata.ui_title_variants">
+        <ng-container *ngFor="let variant_title of record.metadata.ui_title_variants; first as isFirst">
+          <dt class="col-sm-4 offset-sm-2 offset-md-0">
+            <span class="label-title" *ngIf="isFirst; else elseBlock" translate>Variant title</span>
+            <ng-template #elseBlock>&nbsp;</ng-template>
+          </dt>
+          <dd class="col-sm-6 col-md-8 mb-0">
+            {{ variant_title | titlecase }}
+          </dd>
+        </ng-container>
+      </ng-container>
+
+      <!-- RESPONSIBILITY -->
+      <ng-container *ngIf="record.metadata.ui_responsibilities">
+        <ng-container *ngFor="let responsibility of record.metadata.ui_responsibilities; first as isFirst">
+          <dt class="col-sm-4 offset-sm-2 offset-md-0">
+            <span class="label-title" *ngIf="isFirst; else elseBlock" translate>Statement of responsibility</span>
+            <ng-template #elseBlock>&nbsp;</ng-template>
+          </dt>
+          <dd class="col-sm-6 col-md-8 mb-0">
+            {{ responsibility | titlecase }}
+          </dd>
+        </ng-container>
+      </ng-container>
+
       <!-- TYPE -->
-      <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>
+      <dt class="col-sm-4 offset-sm-2 offset-md-0 label-title" translate>
         Type
       </dt>
-      <dd class="col-sm-7 col-md-8 mb-0">
+      <dd class="col-sm-6 col-md-8 mb-0">
         {{ record.metadata.type | translate | titlecase }}
       </dd>
 
       <!-- AUTHORS -->
       <ng-container *ngIf="authors as docAuthors">
         <ng-container *ngIf="docAuthors.length > 0">
-          <dt class="col-sm-3 offset-sm-2 offset-md-0">
+          <dt class="col-sm-4 offset-sm-2 offset-md-0">
             <ng-container *ngIf="docAuthors < 2; else authors" class="label-title" translate>
               Author
             </ng-container>
             <ng-template #authors>
               <span class="label-title" translate>Authors</span>
             </ng-template>
           </dt>
-          <dd class="col-sm-7 col-md-8 mb-0">
```

Comment:
This can be probably set at only one place: a property or a method class

Reply:
Done